<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifierWindow</class>
 <widget class="QWidget" name="NotifierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>160</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="icons" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_icon">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top: 1px solid #A7A7A7;
border-left: 1px solid #A7A7A7;
border-top-left-radius: 4px;
background: #CFCFCF;
padding: 3px;</string>
        </property>
        <property name="text">
         <string notr="true">&lt;img src=&quot;:/img/main.png&quot;/&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_show_hide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
border-left: 1px solid #A7A7A7;
border-bottom: 1px solid #A7A7A7;
border-bottom-left-radius: 4px;
background: #CFCFCF;
padding: 3px;
}
QPushButton:hover {
background: #C0C0C0;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/img/arrow-left-gray.png</normaloff>:/img/arrow-left-gray.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="title" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-top: 1px solid #A7A7A7;
border-right: 1px solid #A7A7A7;
color: black;
padding: 3px;
background: #CFCFCF;</string>
        </property>
        <property name="text">
         <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aspia&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_connections">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
padding: 3px;
border-bottom: 1px solid #A7A7A7;
border-right: 1px solid #A7A7A7;
background: #CFCFCF;</string>
        </property>
        <property name="text">
         <string>Connections</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>128</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="content" native="true">
     <property name="styleSheet">
      <string notr="true">#content {
border: 1px;
border-color: #A7A7A7;
border-left-style: solid;
border-right-style: solid;
border-bottom-style: solid;
border-top-style: none;
background-color:#FAFAFA;
padding: 30px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="tree">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QTreeWidget {
show-decoration-selected: 0;
border: none;
margin: 2px;
}
QTreeWidget::item {
border: 0;
color: #505050;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">0</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
         <property name="textAlignment">
          <set>AlignTrailing|AlignVCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="toolbar" native="true">
        <property name="styleSheet">
         <string notr="true">#toolbar {
background:#FAFAFA;
border: 1px solid #A7A7A7;
}
QToolButton {
border: 1px solid #FAFAFA;
border-radius: 2px;
background:#FAFAFA;
color: #505050;
padding: 2px;
}
QToolButton:hover {
border: 1px solid #AAAAAA;
background: #EBEBEB;
}
QToolButton:pressed {
border: 1px solid #AAAAAA;
background: #D7D7D7;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>194</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="button_text_chat">
           <property name="toolTip">
            <string>Chat</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../client/resources/client.qrc">
             <normaloff>:/img/text-chat.png</normaloff>:/img/text-chat.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_lock_mouse">
           <property name="toolTip">
            <string>Lock mouse</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/mouse-unlock.png</normaloff>:/img/mouse-unlock.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_lock_keyboard">
           <property name="toolTip">
            <string>Lock keyboard</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../client/resources/client.qrc">
             <normaloff>:/img/keyboard.png</normaloff>:/img/keyboard.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_pause">
           <property name="toolTip">
            <string>Pause</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/img/control-pause.png</normaloff>:/img/control-pause.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="button_stop">
           <property name="toolTip">
            <string>Disconnect all</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="icon">
            <iconset resource="../../client/resources/client.qrc">
             <normaloff>:/img/control-stop.png</normaloff>:/img/control-stop.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../client/resources/client.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
